src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -683,8 +700,7 @@ private void processUnprovidedTypes(Set<String> provides, Set<String> transitive
         if (symbolInput != null && symbolInput.isExtern()) continue;
 
         // skip types that come from files that we skip emit on.
```

Comment:
This comment now reads like it's peering into the impl of shouldSkipVar.  I'd say maybe just drop it.